okapi-core/src/main/java/org/folio/okapi/util/TenantInstallOptions.java

```diff
@@ -69,12 +69,23 @@ public boolean getDepCheck() {
     return depCheck;
   }
 
-  public void setInvoke(boolean v) {
+  public void setInvoke(String v) {
     invoke = v;
   }
 
-  public boolean getInvoke() {
-    return invoke;
+  /**
+   * Check if module is to be invoked during install.
+   * @param id module ID.
+   * @return whether to invoke the module.
+   */
+  public boolean checkInvoke(String id) {
+    if (invoke == null || "true".equals(invoke)) {
```

Comment:
Do we want to try and handle "True" / "TRUE" / etc. as well?  `Boolean.parseBoolean(invoke)` might be a better check here?

Reply:
or as suggested below use `String.equalsIgnoreCase(...)` ?

Reply:
The code elsewhere is pretty strict with this .. In ModuleUtil.getParmBoolean where anything but `true` or `false` will be reported as error. So I think we should make it strict to keep the style.

Reply:
Someday, someone will make a `true`module.